cmd/kube-apiserver/app/aggregator.go

```diff
@@ -250,6 +250,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "events.k8s.io", Version: "v1"}:                      {group: 17750, version: 15},
 	{Group: "events.k8s.io", Version: "v1beta1"}:                 {group: 17750, version: 5},
 	{Group: "authentication.k8s.io", Version: "v1"}:              {group: 17700, version: 15},
+	{Group: "authentication.k8s.io", Version: "v1beta1"}:         {group: 17700, version: 5},
```

Comment:
also add to betaAPIGroupVersionsDisabledByDefault in pkg/controlplane/instance.go (will require regenerating openapi doc)

Reply:
Added! thanks. I was searching for this.